content/tutorials/getting-started/go.md

```diff
@@ -15,26 +15,26 @@ messages between them.
 ## Install Go
 
 go-libp2p recommends the use of a Go version that includes the [modules feature](https://github.com/golang/go/wiki/Modules),
```

Comment:
Every Go version you'd realistically use nowadays has modules support. We should remove this sentence.

Reply:
added suggestion